astronomer/providers/amazon/aws/example_dags/example_batch.py

```diff
@@ -0,0 +1,42 @@
+"""Example DAG for the  AWS Batch Operator Async"""
+import os
+from datetime import datetime
+from json import loads
+from os import environ
+
+from airflow import DAG
+
+from astronomer.providers.amazon.aws.operators.batch import BatchOperatorAsync
+
+AWS_DEFAULT_REGION = os.getenv("AWS_DEFAULT_REGION", "us-east-2")
+AWS_CONN_ID = os.getenv("ASTRO_AWS_CONN_ID", "aws_default")
+
+# The inputs below are required for the submit batch example DAG.
+JOB_NAME = environ.get("BATCH_JOB_NAME", "test_airflow_job")
+JOB_DEFINITION = environ.get("BATCH_JOB_DEFINITION", "providers_team_job_defn:1")
+JOB_QUEUE = environ.get("BATCH_JOB_QUEUE", "providers_team_job_queue_ec2")
+JOB_OVERRIDES = loads(environ.get("BATCH_JOB_OVERRIDES", "{}"))
+
+# An existing (externally triggered) job id is required for the sensor example DAG.
+JOB_ID = environ.get("BATCH_JOB_ID", "00000000-0000-0000-0000-000000000000")
+
+
+with DAG(
+    dag_id="example_batch_submit_job",
+    schedule_interval=None,
+    start_date=datetime(2021, 1, 1),
+    tags=["example"],
```

Comment:
```suggestion
    tags=["example", "async", "AWS"],
```

Reply:
done in [f8c026d](https://github.com/astronomer/astronomer-providers/pull/404/commits/f8c026d9b7dccc417b17fd158282d0b265290ecf)